tema1/schedule.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Schedule</title>
+</head>
+
+<body>
+  <header>
+    <h1>My schedule</h1>
+    <label for="time">April 2024</label>
+    <meter id="time" value="0.7" style="width:70%"">Percentage</meter><br>
+    <nav>
+      <a href="/index.html" target="_blank">Home</a> |
+      <a href="/bio.html" target="_blank">Bio</a> |
+      <a href="#" target="_blank">Schedule</a>
+    </nav>
+  </header>
+  <main>
+    <table border="2" style="width:70%">
+      <thead>
+        <tr>
+          <th></th>
+          <th>
+            <h4>Monday</h4>
+          </th>
+          <th>
+            <h4>Tuesday</h4>
+          </th>
+          <th>
+            <h4>Wednesday</h4>
+          </th>
+          <th>
+            <h4>Thursday</h4>
+          </th>
+          <th>
+            <h4>Friday</h4>
+          </th>
+          <th>
+            <h4>Saturday</h4>
+          </th>
+          <th>
+            <h4>Sunday</h4>
+          </th>
+        </tr>
+      <tbody>
+        <tr>
+          <td width>
+            <h1>9-12</h1>
+          </td>
+          <td>
+            <details>
+              <summary>Read more</summary>
+              <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+                industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+                scrambled it to make a type specimen book. </p>
+            </details>
+          </td>
+          <td><img src="https://www.logobird.com/wp-content/uploads/2011/01/html5.png" alt="HTML5" width="100"
+              height="50"></td>
+          <td>Tenis</td>
+          <td>The "power of two" is 2<sup>n</sup></td>
+          <td></td>
+          <td>
+            <b>Bold text</b><br>
+            <em>Emphasized text</em><br>
+            <del>Deleted text</del>
+          </td>
+          <td style="background-color:rgb(255,183,137)">
+            <p>Free</p>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <h2>12-15</h2>
+          </td>
+          <td><a href="https://ro.wikipedia.org/wiki" target="_blank">Wikipedia</a></td>
+          <td style="background-color:rgb(255,183,137)">Free</td>
+          <td></td>
+          <td>
+            <ul>TO DO:
+              <li>cleaning</li>
+              <li>studying</li>
+              <li>cooking</li>
+            </ul>
```

Comment:
Nu poti adauga Text direct intr-un element <ul>
```suggestion
            <p>TO DO:</p>
            <ul>
              <li>cleaning</li>
              <li>studying</li>
              <li>cooking</li>
            </ul>
```